handler/schedule.go

```diff
@@ -8,7 +8,10 @@ import (
 	"github.com/vshn/k8up/job"
 	"github.com/vshn/k8up/scheduler"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"strings"
```

Comment:
Not sure about the code conventions for this projects, but I usually ensure imports from the standard lib are at the top and that they are sorted alphabetically within their respective block.

Reply:
there's no convention in this project. Whatever the IDE creates (in my case Goland), which would be accepted by go fmt, is fine.
would that be configurable in editorconfig somehow?

Reply:
I prefer also to have the suggested order. Goland does this if the standard imports and the other imports are separated by an empty line. But in the end, what ever `go fmt` accepts.

Reply:
I rarely have to tinker manually in the imports, I let the IDE sort and do stuff there. There are now no empty lines and the IDE still doesn't do any sort of ordering. I would like to avoid having to sort manually, I don't think it adds much value and makes maintenance cumbersome.